airbyte-server/src/main/java/io/airbyte/server/handlers/DestinationDefinitionsHandler.java

```diff
@@ -197,6 +200,8 @@ private StandardDestinationDefinition destinationDefinitionFromCreate(final Dest
         destinationDefCreate.getDockerRepository(),
         destinationDefCreate.getDockerImageTag());
 
+    final AirbyteVersion airbyteProtocolVersion = AirbyteProtocolVersion.getWithDefault(spec.getProtocolVersion());
```

Comment:
Not directly related to this PR but AirbyteVersion is confusing. Should we rename it SemanticVersion because it might not be an AirbyteVersion but a protocolVersion instead.

Reply:
Agree, I was thinking of using a 3rd party Version implementation (I saw maven/jackson) until I found out that we had this one already. The slight difference is that our implementation allows the version to be dev.
Happy to submit a PR follow up PR to rename it.